README.md

```diff
@@ -2,8 +2,45 @@
 
 RAPIDS - combined conda package for all of RAPIDS libraries
 
-## RAPIDS Meta-packages
+## RAPIDS Metapackages
 
-The conda recipe in the `conda` folder provides the RAPIDS meta-packages, which when installed will provide the latest RAPIDS libraries for the given version.
+The conda recipe in the `conda` folder provides the RAPIDS metapackages, which when installed will provide the latest RAPIDS libraries for the given version.
 
-See the [README](conda/recipes/README.md) for more information about the meta-packages and how to update versions.
+See the [README](conda/recipes/README.md) for more information about the metapackages and how to update versions.
+
+## dask Metapackage
+
+This repository provides metapackages for pip and conda that centralize the dask version dependency across RAPIDS.
+dask's API instability means that each RAPIDS release must pin to a very specific dask release to avoid incompatibilities.
+These metapackages provide a centralized, versioned storehouse for that pinning.
+The `rapids_dask_dependency` encodes both `dask` and `distributed` requirements.
+
+### Metapackage Versioning
+
+The package is versioned by adding an extra release segment to the standard RAPIDS CalVer.
+For example, the initial release of the metapackage for 23.10.00 will be 23.10.00.00
+This version of the metapackage will always pull the latest nightlies for the dask and distributed packages.
+RAPIDS repos will only pin up to the RAPIDS patch version, i.e. `==23.10.00.*`.
+
+When RAPIDS hits code freeze and we pin dask versions, the package versions in this repository should be pinned.
+At this time, a new metapackage will be released, 23.10.00.1.
+This new metapackage version will be automatically picked up by other RAPIDS libraries since they will be using a `==23.10.00.*` pin.
```

Comment:
Curious whether marking the release as an `alpha` or similar might simplify this release process. Then at release time we could take off the `alpha` and pin

For example with the `dask` nightly conda packages today, we using versioning similar to RAPIDS (like `dask-2023.8.1a230817`)

Reply:
The difficulty here will be that pip installs do not pull in pre-releases without either `pip install --pre` or including the alpha tag in the version spec in pyproject.toml. We are planning to migrate RAPIDS wheels to use alphas in the near future, so if we wanted to we could wait on that effort to reach completion before switching to this metapackage. If we did that then we could use alphas here and remove the extra release segment.

Reply:
FWIW the environment variable `PIP_PRE` works

```bash
$ PIP_PRE=True pip install --dry-run numpy
Collecting numpy
  Obtaining dependency information for numpy from https://files.pythonhosted.org/packages/7f/90/e7829187ad11ea998c26ff69f5581ee7ccaa04f00954829535870c94b07e/numpy-1.26.0b1-cp311-cp311-macosx_11_0_arm64.whl.metadata
  Using cached numpy-1.26.0b1-cp311-cp311-macosx_11_0_arm64.whl.metadata (99 kB)
Using cached numpy-1.26.0b1-cp311-cp311-macosx_11_0_arm64.whl (14.7 MB)
Would install numpy-1.26.0b1
```

<br>

This also works with a `pip.conf`:

```conf
[install]
pre = true
```

```bash
$ pip install --dry-run numpy             
Collecting numpy
  Obtaining dependency information for numpy from https://files.pythonhosted.org/packages/7f/90/e7829187ad11ea998c26ff69f5581ee7ccaa04f00954829535870c94b07e/numpy-1.26.0b1-cp311-cp311-macosx_11_0_arm64.whl.metadata
  Using cached numpy-1.26.0b1-cp311-cp311-macosx_11_0_arm64.whl.metadata (99 kB)
Using cached numpy-1.26.0b1-cp311-cp311-macosx_11_0_arm64.whl (14.7 MB)
Would install numpy-1.26.0b1
```

<br>

Anyways we have some options for configuring `pip` that don't involve altering the command line call directly if that is the concern

<hr>

It's worth noting that conda nightlies for `dask` already need the channel/label `dask/label/dev` added. So there is already a need for passing this alternative options in somehow (this can also be done by altering `condarc` though)

Reply:
The issue isn't around modifying the command line, it's around the fact that `--pre` automatically propagates to all dependencies. If we can avoid handing users a footgun that could install arbitrary alpha versions into an environment, we should. That being said, the work we're now doing to revamp RAPIDS package versioning should help here.

Reply:
With the other changes in RAPIDS using alphas here should now be feasible.